morphir/runtime/src/org/finos/morphir/runtime/quick/GatherReferences.scala

```diff
@@ -13,10 +13,9 @@ import org.finos.morphir.runtime.SDKValue
 
 import scala.collection.immutable.Set
 import zio.Chunk
-import org.finos.morphir.runtime.TypedMorphirRuntimeDefs.{TypeAttribs, ValueAttribs}
 
 object GatherReferences {
-  type TypedValue = Value[TypeAttribs, ValueAttribs]
+  type TypedValue = Value[scala.Unit, UType]
```

Comment:
@edwardpeters Does this line need to change? I am unsure of this one -- should I make it point to `Value.scala` 's typedValue instead? Like
import org.finos.morphir.ir.TypedValue as TV
`type TypedValue = TV`

Reply:
I would point it at `Value.scala`, and maybe skip the rename and just refer to everything as `TypedValue` directly.